public_html/wp-content/themes/pattern-directory/src/components/pattern-order-select/index.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Wordpress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { SelectControl } from '@wordpress/components';
+import { useViewportMatch } from '@wordpress/compose';
+import { addQueryArgs, getQueryArg } from '@wordpress/url';
+
+/**
+ * Internal dependencies
+ */
+import { useRoute } from '../../hooks';
+
+const PatternOrderSelect = ( { options } ) => {
+	const { path, replace } = useRoute();
+	const hideLabel = useViewportMatch( 'medium', '>=' );
+
+	if ( ! options ) {
+		return null;
+	}
+
+	return (
+		<div className="pattern-select-control">
```

Comment:
This should use the same name as the component/folder, so either the class should be `pattern-order-select` or the component should be `pattern-select-control` (but that sounds like you're selecting a pattern).